### PR TITLE
Me/record field license

### PR DESCRIPTION
**Pull request by LHBruneton-C2C**
### Description

This PR introduces support of the licenses field in the record form.

### Quality Assurance Checklist

- [X] Commit history is devoid of any _merge commits_ and readable to facilitate reviews
- [X] If **new logic** ⚙️ is introduced: unit tests were added
- [ ] If **new user stories** 🤏 are introduced: E2E tests were added
- [ ] If **new UI components** 🕹️ are introduced: corresponding stories in Storybook were created
- [ ] If **breaking changes** 🪚 are introduced: add the `breaking change` label
- [ ] If **bugs** 🐞 are fixed: add the `backport <release branch>` label
- [ ] The [documentation website](docs) 📚 has received the love it deserves

### CODE REVIEW

File: libs/feature/editor/src/lib/fields.config.ts [hunk 1]
```diff
@@ -32,4 +32,11 @@ export const DEFAULT_FIELDS: EditorFieldsConfig = [
     },
     onSaveProcess: '${dateNow()}',
   },
+  {
+    model: 'licenses',
+    formFieldConfig: {
+      labelKey: 'editor.record.form.license',
```
Comment (LHBruneton-C2C) [inline libs/feature/editor/src/lib/fields.config.ts:38]: TODO: marker + i18n

File: libs/feature/editor/src/lib/components/record-form/form-field/form-field-license/form-field-license.component.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<gn-ui-dropdown-selector
+  title="license"
```
Comment (LHBruneton-C2C) [inline libs/feature/editor/src/lib/components/record-form/form-field/form-field-license/form-field-license.component.html:2]: TODO: i18n

**Review by jahow — APPROVED** [review]
It looks really good! I think the model for the field configuration might not be fitting anymore, since `formFieldConfig` expects a "type" but the field shown is now more dependant on the `model` property instead.

This can be sorted out later on though. Thanks a lot!

**Comment by LHBruneton-C2C** [comment]
> It looks really good! I think the model for the field configuration might not be fitting anymore, since `formFieldConfig` expects a "type" but the field shown is now more dependant on the `model` property instead.
> 
> This can be sorted out later on though. Thanks a lot!

Yes, there will be some clean up to do, but it will be easier to deal with when no other field relies on it.

**Pull request merged**